server/repositories/events-repository.ts

```diff
@@ -8,17 +10,19 @@ export type EventsRepository = {
     EVENT_ID: string,
     space: string,
     timestamp: number
-  ) => Promise<void>,
+  ) => Promise<void>;
   insertStartedProposal: (
     EVENT_ID: string,
     space: string,
     timestamp: number
-  ) => Promise<void>
+  ) => Promise<void>;
   insertProposalEnd: (
     EVENT_ID: string,
     space: string,
     timestamp: number
-  ) => Promise<void>
+  ) => Promise<void>;
 };
 
-export const eventsRepository: EventsRepository = postgresEventsRepository;
+export const eventsRepository: EventsRepository = useMongoPersistence
```

Comment:
This has me thinking. Since [`gcf-snapshot-analytics`](https://github.com/openlawteam/gcf-snapshot-analytics) uses the Postgres database directly to write queries against can we write to both repositories for a little while until I can get the queries migrated?